modules/distribution/src/repository/resources/dbscripts/apimgt/postgresql.sql

```diff
@@ -126,18 +128,18 @@ CREATE TABLE IF NOT EXISTS IDN_OAUTH2_RESOURCE_SCOPE (
             SCOPE_ID INTEGER NOT NULL,
             TENANT_ID INTEGER DEFAULT -1,
             PRIMARY KEY (RESOURCE_PATH),
-            FOREIGN KEY (SCOPE_ID) REFERENCES IDN_OAUTH2_SCOPE (SCOPE_ID)
+            FOREIGN KEY (SCOPE_ID) REFERENCES IDN_OAUTH2_SCOPE (SCOPE_ID) ON DELETE CASCADE ON UPDATE CASCADE
 );
 
 DROP TABLE IF EXISTS IDN_SCIM_GROUP;
 DROP SEQUENCE IF EXISTS IDN_SCIM_GROUP_PK_SEQ;
 CREATE SEQUENCE IDN_SCIM_GROUP_PK_SEQ;
 CREATE TABLE IDN_SCIM_GROUP (
-                  ID INTEGER DEFAULT NEXTVAL('IDN_SCIM_GROUP_PK_SEQ'),
-                  TENANT_ID INTEGER NOT NULL,
-                  ROLE_NAME VARCHAR(255) NOT NULL,
+			ID INTEGER DEFAULT NEXTVAL('IDN_SCIM_GROUP_PK_SEQ'),
+			TENANT_ID INTEGER NOT NULL,
+			ROLE_NAME VARCHAR(255) NOT NULL,
             ATTR_NAME VARCHAR(1024) NOT NULL,
-                  ATTR_VALUE VARCHAR(1024),
+			ATTR_VALUE VARCHAR(1024),
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04